packages/components/addon/components/hds/card/container.js

```diff
@@ -5,6 +5,8 @@ export const DEFAULT_LEVEL = 'base';
 export const DEFAULT_BACKGROUND = 'neutral-primary';
 export const DEFAULT_OVERFLOW = 'hidden';
 export const LEVELS = ['base', 'mid', 'high'];
+export const HOVER_LEVELS = ['base', 'mid', 'high'];
+export const ACTIVE_LEVELS = ['base', 'mid', 'high'];
```

Comment:
What is the rationale for re-defining levels for each state? Is it so we can potentially restrict the levels we allow for each state? Or do we foresee different levels for different states?

Reply:
As per @heatherlarsen feedback, for now we keep the same three levels for all of them, so I have unified the arrays in a single one.